packages/devtools_app/lib/src/screens/network/network_request_inspector_views.dart

```diff
@@ -163,28 +174,39 @@ class HttpResponseView extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    Widget child;
-    final theme = Theme.of(context);
-    // We shouldn't try and display an image response view when using the
-    // timeline profiler since it's possible for response body data to get
-    // dropped.
-    final contentType = data.contentType;
-    final responseBody = data.responseBody!;
-    if (contentType != null && contentType.contains('image')) {
-      child = ImageResponseView(data);
-    } else if (contentType != null &&
-        contentType.contains('json') &&
-        responseBody.isNotEmpty) {
-      child = JsonViewer(encodedJson: responseBody);
-    } else {
-      child = Text(
-        responseBody,
-        style: theme.fixedFontStyle,
-      );
-    }
-    return Padding(
-      padding: const EdgeInsets.all(denseSpacing),
-      child: SingleChildScrollView(child: child),
+    return ValueListenableBuilder(
+      valueListenable: data.updateCount,
+      builder: (_, __, ___) {
+        Widget child;
+        final theme = Theme.of(context);
+        // We shouldn't try and display an image response view when using the
```

Comment:
This comment probably isn't relevant anymore. @kenzieschmoll, have we dropped support for scraping HTTP events from the timeline?